src/WooCommerce.php

```diff
@@ -1509,8 +1511,59 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
       !empty($brand) ? $brand . ' | no repricing' : 'no repricing' => __('No Repricing', Plugin::L10N),
       !empty($brand) ? $brand . ' | lower prices only' : 'lower prices only' => __('Lower prices only', Plugin::L10N),
     ];
-    
+
     return $dropdown_values;
   }
 
+  /**
+   * Calculates shipping price + tax
+   *
+   * @param  object $shipping_method
+   * @return string
+   */
+  public static function getTotalShippingPrice(object $shipping_method):string {
```

Comment:
```suggestion
  public static function getTotalShippingPrice():string {
```

Reply:
Hey @creative-andrew, we need the $shipping_method argument in order to get the total shipping price for each element:

`$shipping_price_total_with_symbol = self::getTotalShippingPrice($shipping_method);`

Reply:
Sorry I meant it like this 

`public static function getTotalShippingPrice( $shipping_method):string {`

without the object in front.

Reply:
Sorry I meant it like this 

`public static function getTotalShippingPrice( $shipping_method):string {`

without the object in front.